agent/dockerclient/dockerapi/docker_client.go

```diff
@@ -41,6 +41,7 @@ import (
 	"github.com/aws/amazon-ecs-agent/agent/utils/ttime"
 
 	"github.com/cihub/seelog"
+	"github.com/docker/docker/api/types"
```

Comment:
I think I saw this line in last PR, you can update your base branch and rebase to avoid this.

And it would be awesome if you can rebase instead of merge each time you push PR to the `moby` branch, that will make the commits history clearer.